lib/nolibc/Config.uk

```diff
@@ -12,4 +12,23 @@ if LIBNOLIBC
 			Assertions (`assert()` defined in `<assert.h>`) are mapped to `UK_ASSERT()`.
 			If selected, please note that libc assertions are also removed from the code
 			when assertions are disabled in libukdebug.
+
+	config LIBNOLIBC_SYSLOG
+		bool "Include syslog functions"
+		select LIBUKDEBUG
+		default n
+		help
+			Provide implementation of syslog/openlog/closelog functions which use the
+			ukdebug facility.
+
+	menuconfig LIBNOLIBC_TEST
+		bool "Test nolibc"
+		select LIBNOLIBC_TEST_STRSEP if LIBUKTEST_ALL
+		default n
+
+	if LIBNOLIBC_TEST
+	config LIBNOLIBC_TEST_STRSEP
+		bool "test: strsep()"
+		default n
+	endif
```

Comment:
For libc tests we have lib-libc-test (https://github.com/unikraft/lib-libc-test). So I would only include tests here that challenge functionality that is internal and easier to test directly instead of via the POSIX functions.